spec/support/acceptance/litmus_helpers.rb

```diff
@@ -0,0 +1,74 @@
+require 'puppet_litmus'
+
+class LitmusTarget
+  include PuppetLitmus
+
+  attr_reader :uri
+
+  def initialize(uri)
+    @uri = uri
+  end
+end
+
+module LitmusHelpers
```

Comment:
Rationale for this module is that some of Litmus' methods (e.g. `inventory_hash_from_inventory_file`) should be class-level. So, I think `LitmusHelpers.inventory_hash_from_inventory_file` is more readable than `LitmusHelpers.instance.<...>`.

Reply:
Wouldn't a PR to litmus be a better place for this change? It seems like we should use Litmus'  available syntax and if we think that syntax is deficient we should fix Litmus use the new syntax when it becomes available.

Reply:
Yeah, there's some stuff I'd like to fix in Litmus. Problem is "put up PR" => "get changes reviewed" => "get new Litmus release" will take too long so would prefer to keep this for readability. I can add a "TODO" saying "Remove this once Litmus includes these as class methods".